src/Component/Embed/Embed.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Alchemy\Embed\Embed;
+
+use Alchemy\Phrasea\Application;
+
+class Embed
+{
+    /** @var Application */
+    protected $app;
+
+    public function __construct(Application $app)
+    {
+        $this->app = $app;
+    }
+
+    /**
+     * load user configuration for embed bundle
+     * @return array configuration
+     */
+    public function getConfiguration()
+    {
+        // default configuration:
+        $config = [
+          'video' => [
+            'autoplay' => false,
+            'options' => [],
+            'available-speeds' => [],
+            'player' => 'videojs',
+
+          ],
+          'audio' => [
+            'player' => 'videojs',
+            'options' => [],
+            'available-speeds' => [],
+          ],
+          'document' => [
+            'enable-pdfjs' => true,
+            'player' => 'flexpaper',
+          ]
+        ];
+
+        if (isset($this->app['phraseanet.configuration']['embed_bundle'])) {
+            $userConfig = $this->app['phraseanet.configuration']['embed_bundle'];
+
+            // override default options with users defined:
+            $config = array_replace_recursive($config, $userConfig);
+
+            // apply deprecated configuration keys if exists:
+            if (array_key_exists('video_player', $userConfig)) {
```

Comment:
Why not use a `normalizeUserConfig` method to replace config before calling `array_replace_recursive`?